### PR TITLE
Feature/appriciation details screen

### PR DESCRIPTION
**Pull request by amar-josh**
Objection Modal + API integratoin
Jira : [PEER-33](https://peerly-josh.atlassian.net/browse/PEER-33)

### CODE REVIEW

File: src/app/screens/PeerlyScreens/DetailsScreen/index.tsx [hunk 1]
```diff
@@ -1,33 +1,99 @@
...
-          <Text style={styles.name}>Mangesh Pawar</Text>
-          <Text style={styles.title}>Technical Lead</Text>
+          <Text style={styles.name}>
+            {`${cardDetails.receiver_first_name} ${cardDetails.receiver_last_name} `}
+          </Text>
+          <Text style={styles.title}>{cardDetails.receiver_designation}</Text>
         </View>
 
-        <Text style={styles.tag}>Technical Excellence</Text>
+        <Text style={styles.tag}>{cardDetails.core_value_name}</Text>
```
Comment (Chetan-Satpute-Josh) [inline src/app/screens/PeerlyScreens/DetailsScreen/index.tsx:94]: lets convert fields to camel case before using it in react code.

File: src/app/screens/PeerlyScreens/DetailsScreen/index.tsx [hunk 2]
```diff
@@ -1,33 +1,99 @@
...
-          source={require('../../../../assets/images/profile.png')}
+          source={
+            cardDetails?.receiver_image_url
+              ? {uri: cardDetails.receiver_image_url}
+              : require('../../../../assets/images/profile.png')
+          }
           style={styles.avatar}
         />
       </View>
       <View style={{alignItems: 'center', position: 'absolute', right: 80}}>
```
Comment (Chetan-Satpute-Josh) [inline src/app/screens/PeerlyScreens/DetailsScreen/index.tsx:76]: Please avoid using inline styles. We should use style sheets whenever possible.

Comment (amar-josh) [reply src/app/screens/PeerlyScreens/DetailsScreen/index.tsx:76]: Akshay was working on style part so he added temporary inline style. when the beautification will start we will remove all inline styles.

File: src/app/screens/PeerlyScreens/DetailsScreen/index.tsx (hunk 1 above)
Comment (amar-josh) [reply src/app/screens/PeerlyScreens/DetailsScreen/index.tsx:94]: Sure I will made these changes

**Pull request merged**